pulseapi/entries/views.py

```diff
@@ -4,13 +4,54 @@
 
 import django_filters
 from rest_framework import (filters, status)
-from rest_framework.decorators import detail_route
+from rest_framework.decorators import detail_route, api_view
 from rest_framework.generics import ListCreateAPIView, RetrieveAPIView
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.response import Response
 
 from pulseapi.entries.models import Entry
 from pulseapi.entries.serializers import EntrySerializer
+from pulseapi.users.models import UserFavorites
+
+# FIXME: TODO: Remove the handling for GET, so that only PUT works
```

Comment:
correct. The "real" use would be PUT only, but for testing a GET is way easier to do without writing an XHR test page